src/test/kotlin/it/krzeminski/githubactionstyping/LogicTest.kt

```diff
@@ -78,16 +123,16 @@ class LogicTest : FunSpec({
         }
     }
 
-    test("repo with only top-level action and top-level manifest") {
+    test("repo with only top-level action and no top-level manifest") {
         // When
         val (isValid, report) = validateTypings(
             repoRoot = testRepos.resolve("repo-with-only-top-level-action-and-no-top-level-manifest"),
         )
 
         // Then
         assertSoftly {
-            isValid shouldBe false
-            report shouldBe "No action manifest (action.yml or action.yaml) found!"
+            isValid shouldBe true
```

Comment:
Should be false.

Reply:
Idea: we should enforce at least one manifest for a given repo, not necessarily on the top level.